guardian/amigo
# Revert "Prevent AMIgo from upgrading grub-efi-arm64 packages on Ubuntu 18 (#1037)"

## What does this change?

https://github.com/guardian/amigo/pull/1037 was a temporary workaround. Canonical have released a [new AMI](https://amiable.gutools.co.uk/ami?imageId=ami-0a107aa8329bf3762#ami-usage) which means that this workaround is no longer necessary. I have [updated the base image in `PROD`](https://amigo.gutools.co.uk/base-images/ARM%20ubuntu-bionic%20(18.04%20LTS)) to use the new Source AMI, so this workaround can be removed.

## How to test

I have tested this by:

1. Deploying this change to `CODE`
1. Confirming that a bake fails when the base image uses the old Source AMI
1. Updating the base image to use the new Source AMI
1. Re-running a bake and confirming that it now succeeds
1. Deploying a project using the newly baked AMI to confirm that it actually works

## What is the value of this?

We can tidy up the workaround added in https://github.com/guardian/amigo/pull/1037.

## Have we considered potential risks?

I don't think there are any particular risks associated with this PR.
